sidecarhttp/utils.go

```diff
@@ -0,0 +1,16 @@
+package sidecarhttp
```

Comment:
Ppppplease don't make a file called `utils` :) Also, this is only used in tests, so should not get compiled into the main code, right?

Reply:
Oops, I was trying to pull this out into a separate file, so it can be discovered easier, but I screwed up. Would `http_test.go` do (even though it won't have actual tests for the `http.go` file?